blocks/medium-promo-block/features/medium-promo/default.jsx

```diff
@@ -63,6 +113,10 @@ const MediumPromoItem = ({ customFields, arcSite }) => {
           }
         }
         basic {
+					_id
+					auth {
+						${RESIZER_APP_VERSION}
+					}
           type
           url
           resized_params {
```

Comment:
We can remove all references to `resized_params` in the filter as these are no longer used anywhere

Reply:
@matthewroach : Thanks! Removed.